backend/src/main/java/com/bakdata/conquery/models/messages/namespaces/specific/CollectQueryResult.java

```diff
@@ -15,16 +15,26 @@
 /**
  * Workers send their part of the query result to ManagerNode for assembly.
  */
-@CPSType(id="COLLECT_QUERY_RESULT", base=NamespacedMessage.class)
-@AllArgsConstructor @NoArgsConstructor @Getter @Setter @ToString(of = "result")
+@CPSType(id = "COLLECT_QUERY_RESULT", base = NamespacedMessage.class)
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Der Kann weg, AllArgs sollte für Jackson reichen und `results` sollte nie null sein.

Sollte `result.getResults() == null` sein, dann würde der Code noch in ManagedQuery::addResult failen, da ist auch kein null-Check. Der Log kann da aber auch weg